symbulate/random_variables.py

```diff
@@ -5,30 +5,55 @@
 from .utils import is_scalar, is_vector, get_dimension
 
 class RV:
+    """Defines a random variable.
+    
+    A random variable is a function defined on a probability space.
```

Comment:
Replace with "... a function which maps an outcome of a probability space to a number."